### ISSUE TITLE
Error parsing fixed date range in ms from 10.000.000 to 100.000.000

### ISSUE THREAD

**Issue by atais** [grafana/grafana#19738]
**What happened**:

We use grafana to draw chart of data with a strange date frame
`1970-01-01 01:30:00` to `1970-01-01 10:00:00`

So that translates to url 
```&from=1800000&to=32400000```

:smile: 

And it works fine. But if you copy & paste the url into other tab, the `to` date errors as "invalid format". 

**What you expected to happen**:
Graphs should load the date properly

**How to reproduce it (as minimally and precisely as possible)**:
Paste to any chart a parameter `&to=` with values from range 10.000.000 to 100.000.000

**Anything else we need to know?**:
Please don't ask why we use this date-range.

**Environment**:
- Grafana version: 6.2.5
- Data source type & version: MySQL but I don't think it matters
- OS Grafana is installed on: Linux
- User OS & Browser: Linux, Chrome
- Grafana plugins: 
- Others:

**Comment by philrz** [comment]
I ran into this same issue. I'm fine with explaining our use case in case it helps motivate a fix.

We log metrics during automated tests that measure per-second performance and resource utilization of our product under various workloads. Sometimes if we notice a regression or improvement, we'd like to be able to see the metrics compared side-by-side from runs before/after the change. To enable this, we've written a tool that normalizes the timestamps of the per-second points in compared runs so they each start at time=0. Therefore we ran into this issue when we had our first run that went long enough that it stretched beyond a time of `9999999`.
